superdesk/utils.py

```diff
@@ -35,6 +35,32 @@ class SortOrder(Enum):
     desc = 2
 
 
+@unique
```

Comment:
imo this might be limiting on a base class, maybe we can leave this to the concrete implementation?